template/docker-compose.yml

```diff
@@ -1,4 +1,8 @@
+{{#if_eq transporter "AMQP10"}}
+version: "2.1"
```

Comment:
Why need a lower docker-compose version for the health-check?

Reply:
`condition: service_started` is not supported on docker-compose v3